lib/services/rest_api/album_type.dart

```diff
@@ -0,0 +1 @@
+enum AlbumType { calendar, album, folder, moments, state }
```

Comment:
Maybe we could combine the type into a "rest_api_types.dart" file?

Reply:
You mean all enums in one big enum?
I'd prefer seperate files with seperate enums. It's clear what enum is used for which purpose. New values will be added in the future, because these are just the values I found on the first look. 

If you mean something else, please explain.